sheets/prctl

```diff
@@ -6,7 +6,7 @@
 # Examine process limits and permissions:
 prctl ${PID}
 
-# Examine process limits and permissions in machine parseable format:
+# Examine process limits and permissions in machine parse-able format:
```

Comment:
Either is fine, IMO. Think: 'liveable', 'doable', 'driveable', etc. Sometimes these things show up in spelling checkers, but it's fine, and is often even shown as an alternative. Put it this way, I doubt you'll get robbed of a mark in English!